fury/animation/animation.py

```diff
@@ -466,7 +466,7 @@ def set_position_interpolator(self, interpolator, *, is_evaluator=False, **kwarg
         --------
         >>> Animation.set_position_interpolator(spline_interpolator, degree=5) # doctest: +SKIP
 
-        """
+        """  # noqa: E501
```

Comment:
This will go under a different PR. Or update the title a bit to accommodate this. I guess updating the title is enough for this.

Reply:
Thank you, I think I'll open a pr for it.